### PR TITLE
Updated to latest translations

### PR DESCRIPTION
**Pull request by vicbergquist**
Hi 👋
Just wanted to check out your project/locale collection. Love it.
Got the latest translations as well.

Cheers

### CODE REVIEW

**Comment by caugner** [comment]
Hi 👋 

Thank you for the PR and sorry for the delay (I didn't get a notification for some reason, not the first time, will look into this).

File: locales/ar.json [hunk 1]
```diff
@@ -1,689 +1,487 @@
-{
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/ar.json]: Some of the updates reintroduce the outdated key hierarchy (without the `org.vuejs` prefix) which I had fixed manually:

grafik

Those are the ones with a lot of lines added and removed:

grafik

File: locales/id.json [hunk 2]
```diff
@@ -1,689 +1,487 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/id.json]: dito

File: locales/ko.json [hunk 3]
```diff
@@ -1,687 +1,467 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/ko.json]: dito

File: locales/pl.json [hunk 4]
```diff
@@ -1,689 +1,483 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/pl.json]: dito

File: locales/pt.json [hunk 5]
```diff
@@ -1,689 +1,542 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/pt.json]: dito

File: locales/pt_BR.json [hunk 6]
```diff
@@ -1,689 +1,478 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/pt_BR.json]: dito

File: locales/ru.json [hunk 7]
```diff
@@ -1,689 +1,487 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/ru.json]: dito

File: locales/th.json [hunk 8]
```diff
@@ -1,689 +1,487 @@
-{
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/th.json]: dito

File: locales/zh_TW.json [hunk 9]
```diff
@@ -1,689 +1,300 @@
 {
-  "org": {
-    "vue": {
```
Comment (caugner) [inline locales/zh_TW.json]: dito

File: locales/ar.json (hunk 1 above)
Comment (vicbergquist) [reply locales/ar.json]: sorry about that! should be fixed now

**Comment by caugner** [comment]
Hi again,

I have added a json-prettifier now and applied it to the locales to reduce the number of changed lines.
It turned out that the only locales with actual updates were `es`, `ja` and `nb_NO`.

The additions and deletions (mostly deletions) in the following locales stem from the fact, that Transifex only exports keys that are still present in the original language as well as the original values for keys that have not been translated yet:

* `ar`
* `id`
* `ko`
* `pl`
* `pt`
* `pt_BR`
* `ru`
* `th`
* `tr` (change from `CRLF` to `LF` line endings)
* `zh_TW`

Therefore I have reverted the changes to these files for now.

**Pull request merged**